{"1":{"str":"php_pinpoint"},"2":{"str":"PHPApplication"},"3":{"i64":1527751566535},"4":{"str":"ABhwaHBfcGlucG9pbnTHyduouywS"},"7":{"i64":9052620277848141934},"8":{"i64":-1},"9":{"i64":1527751580782},"10":{"i32":0},"11":{"str":"/test_recursive_exception2.php"},"12":{"i16":1500},"13":{"str":"127.0.0.1:8989"},"14":{"str":"127.0.0.1"},"15":{"lst":["rec",1,{"1":{"i32":999},"2":{"rec":{"1":{"str":"/home/liumingyi/git/pinpoint_c_agent/simulate/testcase/php/trace/test_recursive_exception2"}}}}]},"16":{"i16":0},"17":{"i32":1},"18":{"lst":["rec",4,{"7":{"i64":9052620277848141934},"8":{"i16":3},"9":{"i32":0},"10":{"i32":0},"12":{"i16":1501},"14":{"lst":["rec",1,{"1":{"i32":-1},"2":{"rec":{"1":{"str":"{[0]:0 }"}}}}]},"15":{"i32":4},"16":{"i64":-1},"25":{"i32":-26},"26":{"rec":{"1":{"i32":-1},"2":{"str":"exception: n < 1 in /home/liumingyi/git/pinpoint_c_agent/simulate/web/php/test_recursive_exception2.php on line 7"}}}},{"7":{"i64":9052620277848141934},"8":{"i16":2},"9":{"i32":0},"10":{"i32":0},"12":{"i16":1501},"14":{"lst":["rec",1,{"1":{"i32":-1},"2":{"rec":{"1":{"str":"{[0]:1 }"}}}}]},"15":{"i32":3},"16":{"i64":-1},"25":{"i32":-26}},{"7":{"i64":9052620277848141934},"8":{"i16":1},"9":{"i32":0},"10":{"i32":0},"12":{"i16":1501},"14":{"lst":["rec",1,{"1":{"i32":-1},"2":{"rec":{"1":{"str":"{[0]:2 }"}}}}]},"15":{"i32":2},"16":{"i64":-1},"25":{"i32":-26}},{"7":{"i64":9052620277848141934},"8":{"i16":0},"9":{"i32":0},"10":{"i32":0},"12":{"i16":1501},"14":{"lst":["rec",1,{"1":{"i32":-1},"2":{"rec":{"1":{"str":"{[0]:3 }"}}}}]},"15":{"i32":1},"16":{"i64":-1},"25":{"i32":-26}}]},"25":{"i32":-13},"26":{"rec":{"1":{"i32":-2},"2":{"str":"Fatal error: Uncaught exception 'Exception' with message 'n < 1' in /home/liumingyi/git/pinpoint_c_agent/simulate/web/php/test_recursive_exception2.php:7\nStack trace:\n#0 /home/liumingyi/git/pinpoint_c_agent/simulate/web/php/test_recursive_exception2.php(9): test_cumsum_e2(0)\n#1 /home/liumingyi/git/pinpoint_c_agent/simulate/web/php/test_recursive_exception2.php(9): test_cumsum_e2(1)\n#2 /home/liumingyi/git/pinpoint_c_agent/simulate/web/php/test_recursive_exception2.php(9): test_cumsum_e2(2)\n#3 /home/liumingyi/git/pinpoint_c_agent/simulate/web/php/test_recursive_exception2.php(12): test_cumsum_e2(3)\n#4 {main}\n  thrown in /home/liumingyi/git/pinpoint_c_agent/simulate/web/php/test_recursive_exception2.php on line 7"}}}}